nixos/modules/services/networking/nix-store-gcs-proxy.nix

```diff
@@ -0,0 +1,74 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  opts = { name, config, ... }: {
+    options = {
+      enable = mkOption {
```

Comment:
You probably want to use `mkEnableOption` where. It is basically shortcut for what you have constructed here.

Reply:
AFAIU `mkEnableOption` sets `false` as the default, whereas here we want it to be `true`.

Reply:
fair